src/components/modes/teacher/ImageUpload.tsx

```diff
@@ -107,7 +107,7 @@ const ImageUpload = (): JSX.Element => {
         uppy={uppy}
         locale={{
           strings: {
-            chooseFiles: t('Browse Your Computer'),
+            chooseFiles: t('Browse Your Computer', 'Browse your computer'),
```

Comment:
What is the second argument used for ?

Reply:
It's the default value. Otherwise the type checking throws an error... I couldn't manage to configure it to make sure it would pass *at least* an empty string